### ISSUE TITLE
Easily detach training process from terminal while still being able to follow progress

### ISSUE THREAD

**Issue by matthewdzmura** [pytorch/torchtune#371]
I've been using TorchTune by sshing into a workstation from my Macbook. When I kick off a run e.g.

tune --nnodes 1 --nproc_per_node 1 --config alpaca_llama2_full_finetune

and leave my Macbook, it will close the ssh connection, which kills the training job. To get around this, I've been detaching the job from the terminal so that if I close the connection it will continue running. Unfortunately, when I do this I no longer have an easy way to check progress anymore, or know if the job is done. Ideally the training process would detach from the terminal so closing it doesn't stop the job, but the terminal could still be used to monitor progress and pass along a SIGINT signal (Ctrl C) if I want to manually terminate the job.
